From: Dean Nelson <dcn@sgi.com>
Date: Thu, 2 Oct 2008 17:18:21 +0000 (-0500)
Subject: x86, UV: add uv_setup_irq() and uv_teardown_irq() functions, v3
X-Git-Tag: v2.6.28-rc1~80^2~27
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Ftorvalds%2Flinux-2.6.git;a=commitdiff_plain;h=4173a0e7371ece227559b44943c6fd456ee470d1
References: bnc#442461

x86, UV: add uv_setup_irq() and uv_teardown_irq() functions, v3

Provide a means for UV interrupt MMRs to be setup with the message to be sent
when an MSI is raised.

Signed-off-by: Dean Nelson <dcn@sgi.com>
Signed-off-by: Ingo Molnar <mingo@elte.hu>
Acked-by: Bernhard Walle <bwalle@suse.de>

Automatically created from "patches.fixes/uv_setup_irq.diff" by xen-port-patches.py

Index: head-2008-12-01/arch/x86/kernel/Makefile
===================================================================
--- head-2008-12-01.orig/arch/x86/kernel/Makefile	2008-12-01 11:49:07.000000000 +0100
+++ head-2008-12-01/arch/x86/kernel/Makefile	2008-12-01 12:06:39.000000000 +0100
@@ -126,4 +126,4 @@ endif
 
 disabled-obj-$(CONFIG_XEN) := %_uv.o early-quirks.o hpet.o i8253.o \
 	i8259.o irqinit_$(BITS).o pci-swiotlb_64.o reboot.o smpboot.o \
-	tlb_$(BITS).o tsc.o tsc_sync.o vsmp_64.o
+	tlb_$(BITS).o tsc.o tsc_sync.o uv_%.o vsmp_64.o
